plugins/org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/XtextInjectorProvider.java

```diff
@@ -8,22 +8,22 @@
 
 import com.google.inject.Injector;
 
-public class XtextInjectorProvider implements IInjectorProvider, IRegistryConfigurator
-{
+/**
+ * @since 2.9
+ */
+public class XtextInjectorProvider implements IInjectorProvider, IRegistryConfigurator {
```

Comment:
This injector provider is used to test Xtext itself. I don't think it should be part of the juni4 test library.